modules/core/src/test/java/org/apache/ignite/testframework/junits/JUnitAssertAware.java

```diff
@@ -203,4 +203,33 @@ protected static void assertNotSame(Object unexpected, Object actual) {
     protected static void assertNotSame(String msg, Object exp, Object actual) {
         Assert.assertNotSame(msg, exp, actual);
     }
+
+    /** Check arrays equality as well as objects equality. */
+    protected static void assertEqualsArraysOrObjects(Object exp, Object actual) {
+        assertEqualsArraysOrObjects(null, exp, actual);
+    }
+
+    /** Check arrays equality as well as objects equality. */
+    protected static void assertEqualsArraysOrObjects(String msg, Object exp, Object actual) {
```

Comment:
We have copy of this method - `FunctionalTest#assertEqualsArraysAware` - let's keep one copy and deduplicate.